pkg/controller/windowsmachineconfig/status.go

```diff
@@ -62,6 +62,24 @@ func (s *StatusManager) updateStatus() error {
 	return nil
 }
 
+// updateStatusReconciling updates the status of the WMC to indicate that
+// it is reconciling
+func (s *StatusManager) updateStatusReconciling() error {
```

Comment:
Why to have a separate method for reconciling, why can't it be part of `updateStatus` method?

Reply:
this can be a part of update status method, just wanted to avoid unnecessary setting of status fields for object when they are not set in the struct, but I can add validations in the updateStatus method to set the values only if they are set in the struct